pycape/cape.py

```diff
@@ -387,6 +437,47 @@ async def _request_connection(self, function_ref, pcrs=None):
                 )
         return
 
+    async def _close(self):
+        await self._ctx.close()
+
+    async def _deploy(self, deploy_path):
+        deploy_path = pathlib.Path(deploy_path)
+        cmd_deploy = "cape deploy " + str(deploy_path)
```

Comment:
probably want to include the URL that was passed to `Cape` client here with a CLI flag `-u {self.url}`

Reply:
good point